.github/workflows/rust-test.yml

```diff
@@ -17,6 +17,7 @@ jobs:
         with:
           submodules: recursive
       - uses: dtolnay/rust-toolchain@stable
+      - uses: Swatinem/rust-cache@v2
```

Comment:
Is this change related to coverage?

Reply:
It's not directly, but running the coverage action was taking quite some time. I added this to all files at the same time